tests/test_MatchPessimisticB.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
```

Comment:
In that case someone better start cleaning up meas_astrom fast because the majority of the unittests have #!. This there currently because the test was copied from MatchOptimisticB. It is now removed.